plugins/modules/win_acl.ps1

```diff
@@ -116,6 +116,9 @@ while ($follow) {
 
     if ($link_info -and $link_info.Type -in @("SymbolicLink", "JunctionPoint")) {
         $path = $link_info.AbsolutePath
+        if ($link_info.Type -eq "JunctionPoint") {
```

Comment:
I'm not sure this is the correct check to do here. While a junction point might refer directly to an NT device/volume it can still be used to point to a folder like a symbolic link. It might be worthwhile to check `$link_info.SubstituteName` which is the raw path of the junction point target and see if it's in the `\??\Volume{guid}` format. It could also check if the `$path` starts with `Volume{guid}` pattern.

Only then should the new code here prefix the `AbsolutePath` with `\\?\`.

Reply:
Note for info:
`$link_info.SubstituteName = \??\Volume{f7ae69d2-76fe-4650-b4ba-9afab7c8dfeb}\`
`$path = Volume{f7ae69d2-76fe-4650-b4ba-9afab7c8dfeb}\`

I don't know yet what is the best approach to be more generic as possible